### PR TITLE
add support for building multi-arch docker images

### PR DESCRIPTION
**Pull request by rparmer**
Update Dockerfiles and pipelines to build multi-arch variants of images

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -66,7 +66,7 @@ jobs:
           builder: ${{ steps.buildx.outputs.name }}
           context: .
           file: ./Dockerfile
-          platforms: linux/amd64 #,linux/arm/v7,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```
Comment (chanwit) [inline .github/workflows/release.yaml:69]: Please remove arm/v7 for the moment.
Production clusters would use only amd64 and arm64.

```suggestion
          platforms: linux/amd64,linux/arm64 # linux/arm/v7
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -85,7 +85,7 @@ jobs:
           builder: ${{ steps.buildx.outputs.name }}
           context: .
           file: ./runner.Dockerfile
-          platforms: linux/amd64 #,linux/arm/v7,linux/arm64
+          platforms: linux/amd64,linux/arm64,linux/arm/v7
```
Comment (chanwit) [inline .github/workflows/release.yaml:88]:
```suggestion
          platforms: linux/amd64,linux/arm64 # linux/arm/v7
```

**Review by chanwit — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yaml (hunk 2 above)
Comment (rparmer) [reply .github/workflows/release.yaml]: done

**Review by chanwit — APPROVED** [review]
Thank you so much Russ!

**Pull request merged**